.ruby-version

```diff
@@ -1 +1,3 @@
-2.2.3
+2.1.7
+# DO NOT UPGRADE THIS RUBY VERSION
```

Comment:
It can be upgraded to 2.1.x, just not 2.2.x. What about "Don't upgrade to 2.2.x until the gotcha in Govspeak with URI in Ruby 2.2.x is resolved. See: https://github.com/alphagov/whitehall/pull/2234"?